### ISSUE TITLE
Samsung tts vs Google tts

### ISSUE THREAD

**Issue by Abdulrahman392011** [JHubi1/ollama-app#60]
### Requirements

- [X] I have searched the issues of this repository and believe that this is not a duplicate
- [X] I have confirmed this bug exists on the latest version of the app

### Platform

Android

### Description

Tts how to

### Steps to reproduce

In samsung phones the default tts engine is samsung not google. You must change it to google for tts to work in the app

### Expected behavior

.

### Actual behavior

.

**Comment by JHubi1** [comment]
You closed the issue; is the problem solved?

**Comment by Abdulrahman392011** [comment]
No, its not worth your time. I was just mentioning it so you can make a not in the app for people that use Samsung or other tts engines to notice that they need to switch to google tts engine for the voice feature to work.

**Comment by JHubi1** [comment]
May revisit this in the future
